org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/src/main/java/org/eclipse/che/jdt/ls/extension/core/internal/ChangeUtil.java

```diff
@@ -116,8 +112,18 @@ public static void convertChanges(Change change, CheWorkspaceEdit edit) throws C
     return result;
   }
 
-  private static void convertCompositeChange(Change change, CheWorkspaceEdit edit)
-      throws CoreException {
+  private static void convertCompositeChange(
```

Comment:
If the method is called "convertCompositeChange" it should take a "CompositeChange". Do the type dispatching in one method and when you know the type, dispatch to typesafe methods.

Reply:
fixed